compiler_opt/rl/problem_configuration.py

```diff
@@ -105,13 +106,14 @@ def get_runner_type(self) -> 'type[compilation_runner.CompilationRunner]':
 
   # List of flags to add to clang compilation command. The flag names should
   # match the actual flags provided to clang. An example for AFDO reinjection:
-  # return ['-fprofile-sample-use=/path/to/gwp.afdo',
-  #  '-fprofile-remapping-file=/path/to/prof_remap.txt']
-  def flags_to_add(self) -> Tuple[str, ...]:
-    return ()
+  # return ('-fprofile-sample-use=/path/to/gwp.afdo',
+  #  '-fprofile-remapping-file=/path/to/prof_remap.txt')
+  @gin.configurable(module='problem_config')
```

Comment:
is it needed? can you just use ProblemConfiguration.flags_to_add.add_flags?

Reply:
i tried ProblemConfiguration.flags_to_add.add_flags by making ProblemConfiguration @gin.configurable but that didn't work, got an error saying 
`ValueError: No configuration matching ProblemConfiguration.flags_to_add.add_flags`

Reply:
does InliningConfig.flags_to_add.add_flags work?

Reply:
It doesn't work. I get an error saying 
`ValueError: No configuration matching InliningConfig.flags_to_add.add_flags`